website/profile/utils.py

```diff
@@ -33,6 +33,8 @@ def serialize_user(user, node=None, admin=False, full=False, is_profile=False, i
         'surname': user.family_name,
         'fullname': fullname,
         'shortname': fullname if len(fullname) < 50 else fullname[:23] + '...' + fullname[-23:],
+        'is_institutional_admin': user.is_institutional_admin(),
```

Comment:
Do we need this for this view? It looks like it's being used, but is it being used properly? See below for more.